markdown/boomi-v2-getting-started/boomi-v2-getting-started.md

```diff
@@ -234,39 +214,44 @@ Positive
 
 ![canvas](img/canvas.png "canvas")
 
-### 2.. Configure Solace Shape
-👉 Hover over the Start shape to configure it 
+### 2. Search for the Solace shape from 
```

Comment:
not sure if you meant to say from where?

Reply:
yea just fixed that as well, wanted to say from the list of shapes 👍